src/components/SideNavBar/__tests__/SideNavBar.test.tsx

```diff
@@ -20,93 +20,71 @@ describe('SideNavBar', () => {
     expect(activeLinks && activeLinks.length).toBe(1);
   });
 
-  it('should set the body overflow hidden on menu click', () => {
-    render(<SideNavBar pageKey={SideNavBarKeys.releases} />);
-    const downloadItem: Element = screen.getAllByRole('button')[0] as Element;
-    expect(document.body.style.overflow).toBe(OverflowTypes.unset);
-    fireEvent.click(downloadItem);
-    expect(document.body.style.overflow).toBe(OverflowTypes.hidden);
-  });
-
-  it('should set the body overflow hidden/unset on toggling', () => {
-    render(<SideNavBar pageKey={SideNavBarKeys.releases} />);
-    const downloadItem: Element = screen.getAllByRole('button')[0] as Element;
-    expect(document.body.style.overflow).toBe(OverflowTypes.unset);
-    fireEvent.click(downloadItem);
-    expect(document.body.style.overflow).toBe(OverflowTypes.hidden);
-    fireEvent.click(downloadItem);
-    expect(document.body.style.overflow).toBe(OverflowTypes.unset);
-    fireEvent.click(downloadItem);
-    expect(document.body.style.overflow).toBe(OverflowTypes.hidden);
-    fireEvent.click(downloadItem);
-    expect(document.body.style.overflow).toBe(OverflowTypes.unset);
-  });
-
   it('should contain a href to `~/about`', () => {
     render(<SideNavBar pageKey={SideNavBarKeys.about} />);
     const aboutNavBarElement = screen.getByText('About');
-    expect(aboutNavBarElement.getAttribute('href')).toBe('/about');
+    expect(aboutNavBarElement.getAttribute('href')).toBe('/about/');
```

Comment:
What's causing these trailing slashes?

Reply:
We need these trailing slashes. Ironically, Gatsby router works as every route is a "directory" (kinda), so for example if I open `https://nodejs.dev/about` it will have a trailing slash.

But if I use the `Navigation` component or `SideNavBar` component, it was not adding that trailing slash. So, for example, the built-in IntersectionObserver would not work. (Looks at least that for Chrome, if both routes match, and if I refresh the page or close and reopen the tab it already goes back to where I was reading)